pallets/omnipool/src/lib.rs

```diff
@@ -468,8 +468,11 @@ pub mod pallet {
 
 			let amount = T::Currency::free_balance(asset, &Self::protocol_account());
 
+			let ed = T::AssetRegistry::existential_deposit(asset);
+			let minimum_pool_liquidity = ed.checked_mul(20).ok_or(ArithmeticError::Overflow)?;
```

Comment:
saturating_mul is enough here.

if the max value is reached, you would not be able to have a balance anyway.